core/domain/learner_playlist_services_test.py

```diff
@@ -257,6 +298,46 @@ def test_mark_collection_to_be_played_later(self):
             self._get_all_learner_playlist_collection_ids(
                 self.user_id), collection_ids)
 
+        # Empty the learner playlist again.
```

Comment:
Same question -- a lot of this seems to be copying stuff from above. What is the rationale behind this?

Reply:
Same as above!

Reply:
Ditto! (See https://github.com/oppia/oppia/pull/6291#discussion_r257555996)

Reply:
Also done!